### PR TITLE
Zurich/twirf/1

### PR DESCRIPTION
**Pull request by dns2utf8**
Preparation for publication:

- [x] rename file to fit the schema: 2017-07-06-this-week-in-rustfest-1-contest-and-cfp

### CODE REVIEW

**Comment by Hoverbear** [comment]
I'd like to do a teaser of things we're working on with our other friends in the community! :) Proposed patch coming.

**Comment by Hoverbear** [comment]
How's that?

File: _posts/2017-07-06-TWIRF-contest-and-cfp.md [hunk 1]
```diff
@@ -20,8 +20,16 @@ Some ideas to talk about:
...
 
+## Stories from the Campfire
+
+We've resumed our weekly meetings (often spanning several timezones and countries) to bring together this event. Back in Berlin, we've been drawing together the plans for workshops with our old friends [Parity](https://parity.io/). Also in Berlin, our friends at [1aim](https://1aim.com) have been putting together a contest to help with the libz blitz (more on that below). Over in America, some of our organizers have been in touch with (and part of) the team at [Rust Belt Rust](http://rust-belt-rust.com/) to support each other and learn each others lessons. Across the globe we've been coordinating with various [Mozillians](http://mozilla.org) as well as community members to support our attendees and speakers.
+
+Local, in Zürich, we've been exploring (delicious) catering options. On campus, we've been collaborating with [ETH](https://www.ethz.ch/) to create an amazing atmosphere and setting for the event. We've even been thinking about offering screenprinting, [let us know if you're interested.](https://github.com/RustFestEU/blog.rustfest.eu/issues/16). We can't wait to share this event with you!
+
+We're so proud of the
```
Comment (gandro) [inline _posts/2017-07-06-TWIRF-contest-and-cfp.md]: Leftover sentence?

**Comment by gandro** [comment]
Awesome, looks good to me (modulo the stray sentence). Feel free to publish!

**Review by gandro — APPROVED** [review]

File: _posts/2017-07-06-TWIRF-contest-and-cfp.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "TWIRF: CFP Open and Libz Blitz Contest"
```
Comment (dns2utf8) [inline _posts/2017-07-06-TWIRF-contest-and-cfp.md]: Should we call this `TWIRF1: ...` or `This Week in RustFest 1: ...`?

**Pull request merged**